examples/guides/se-restful-webservice/README.adoc

```diff
@@ -40,19 +43,19 @@ You'll learn how to use Helidon quickly to create a RESTful web service that acc
 |`PUT localhost:8080/greet/greeting/Hola` |Changes the greeting used in subsequent responses
 |===
 
-We create the app in three main steps:
+Create the app in three main steps:
 
-. Write a basic Helidon SE app to respond to the HTTP requests.
+. Use the Helidon Maven archetype to create a basic Helidon SE app that responds
+to the HTTP requests.
 
-. Add code to perform simple health checks.
+. Add code to perform a simple app-specific health check.
 
-. Add code to record simple metrics.
+. Add code to record a simple app-specific metric.
 
-This guide walks you through the code and helps you understand what each part of the
-code does as you write the app. But if you prefer, you can get the finished code for this example.
+This guide walks you through the generated and added code and helps you understand what each part of the
+code does. But if you prefer, you can get the finished code for this example.
```

Comment:
I suggest:

"As you develop the app, this guide helps you understand what each part of the code does. If you prefer to download the finished code for this example, follow the instructions at <<downloading,Download the example source>>."

Reply:
Done